CMakeLists.txt

```diff
@@ -7,10 +7,10 @@ set(TARAXA_PATCH_VERSION 1)
 set(TARAXA_VERSION ${TARAXA_MAJOR_VERSION}.${TARAXA_MINOR_VERSION}.${TARAXA_PATCH_VERSION})
 
 # Any time a change in the network protocol is introduced this version should be increased
-set(TARAXA_NET_VERSION 1)
+set(TARAXA_NET_VERSION 2)
 # Major version is modified when DAG blocks, pbft blocks and any basic building blocks of our blockchain is modified
 # in the db
-set(TARAXA_DB_MAJOR_VERSION 1)
+set(TARAXA_DB_MAJOR_VERSION 2)
```

Comment:
If I am not mistaken this change is minor change, not major. So if we can deal with data format change with migration we should change only minor version and migration will be applied. But on major version change we should rebuild whole db